src/runtime_sort.cpp

```
@@ -204,7 +206,7 @@ void __ry_timsort(void *data, int64_t len, int64_t elemSize,
 
     // Allocate temp buffer for merging (max half of array)
     int64_t tmpSize = (len / 2 + 1) * elemSize;
```

Clang (14.0.6)

[warning] variable 'tmpSize' is not initialized

(cppcoreguidelines-init-variables)